opentelemetry-sdk/src/opentelemetry/sdk/_metrics/__init__.py

```diff
@@ -46,95 +62,179 @@ def __init__(
         self._instrumentation_info = instrumentation_info
         self._meter_provider = meter_provider
 
-    def create_counter(self, name, unit=None, description=None) -> Counter:
-        # FIXME implement this method
-        pass
+    def create_counter(self, name, unit=None, description=None) -> APICounter:
+        # pylint: disable=protected-access
+        return self._meter_provider._create_counter(
```

Comment:
I don't get why this has to call into the MeterProvider class. I also agree with pylint's warning here, why do we need to do it this way?

Reply:
I have removed the `_create_*` methods from the `MeterProvider`.